envcov.sh

```diff
@@ -33,7 +33,7 @@ then
 fi
 
 LCOV_PATH="${XCODECOVERAGE_PATH}/lcov-1.11/bin"
-OBJ_DIR=${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}
+OBJ_DIR="${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}"
 
 # Fix for the new LLVM-COV that requires gcov to have a -v paramter
 LCOV() {
```

Comment:
Yeah, the quoting is during the act of setting, not on the set value.  When you then go to use the variable, its value is literally substituted, bringing back the whitespace issue.